### DISCUSSION TITLE
Terminology: Supremacy

### DISCUSSION THREAD

**Discussion by RebelusQuo** [RebelusQuo/the-wall-is-down#6] [Ideas]
The terms are slightly muddled when it comes to the rules around Supremacy.

Some cards talk about gaining Supremacy, either simply or after removing others Supremacy. The base rule is also of the simple kind. However, "gaining" sounds more like the later case where others Supremacy is removed. What about "vying for Supremacy", which would be contrasted by "gain direct Supremacy"?

**Comment by rodrigoSantamaria** [comment]
You're right. Most of the cards, and also the basic ops for the NWO track, would gain supremacy if no one has it, or remove it if somebody else has. So, supremacy is the fact of having the supremacy token in a given NWO track's slot.
Yes, we can use "vying" for normal rules and "gain direct" for special card which grant the supremacy even if others had it previously.

**Comment by RebelusQuo** [comment]
Perfect, I will work on the wording of those kinds of cards and add specific card discussions if needed.
